sui_core/src/authority.rs

```diff
@@ -329,10 +330,13 @@ impl AuthorityState {
 
         // At this point we need to check if any shared objects need locks,
         // and whether they have them.
-        let has_shared_object = objects_by_kind
+        let shared_object: Vec<_> = objects_by_kind
```

Comment:
Should this be called `shared_objects` since we may have many of them?

Reply:
Yes, obviously!